pkg/handlers/get.go

```diff
@@ -1,43 +1,31 @@
 package handlers
 
 import (
+	"encoding/json"
 	"net/http"
 
-	"github.com/Ubivius/microservice-template/pkg/data"
+	"github.com/Ubivius/microservice-matchmaking/pkg/data"
 )
 
-// GetProducts returns the full list of products
-func (productHandler *ProductsHandler) GetProducts(responseWriter http.ResponseWriter, request *http.Request) {
-	productHandler.logger.Println("Handle GET products")
-	productList := data.GetProducts()
-	err := data.ToJSON(productList, responseWriter)
+// GetQueue returns the full list of players in queue
+func (queueHandler *QueueHandler) GetQueue(responseWriter http.ResponseWriter, request *http.Request) {
```

Comment:
Do we need any get functions on the queue apart from testing purposes? If its only for testing they should not be handlers and instead individual test functions on business logic